packages/client/scripts/postinstall.js

```diff
@@ -197,30 +194,49 @@ function run(cmd, params, cwd = process.cwd()) {
   })
 }
 
-async function ensureEmptyDotPrisma() {
+/**
+ * Copies our default "throw" files into the default generation folder. These
+ * files are dummy and informative because they just throw an error to let the
+ * user know that they have forgotten to run `prisma generate`.
+ */
+async function createDefaultGeneratedThrowFiles() {
   try {
     const dotPrismaClientDir = path.join(__dirname, '../../../.prisma/client')
+    const defaultNodeIndexPath = path.join(dotPrismaClientDir, 'index.js')
+    const defaultBrowserIndexPath = path.join(dotPrismaClientDir, 'index-browser.js')
+    const defaultNodeIndexDtsPath = path.join(dotPrismaClientDir, 'index.d.ts')
     await makeDir(dotPrismaClientDir)
-    const defaultIndexJsPath = path.join(dotPrismaClientDir, 'index.js')
-    const defaultIndexBrowserJSPath = path.join(dotPrismaClientDir, 'index-browser.js')
-    const defaultIndexDTSPath = path.join(dotPrismaClientDir, 'index.d.ts')
 
-    if (!fs.existsSync(defaultIndexJsPath)) {
-      await copyFile(path.join(__dirname, 'default-index.js'), defaultIndexJsPath)
+    const dotPrismaClientEdgeDir = path.join(dotPrismaClientDir, 'edge')
+    const defaultEdgeIndexPath = path.join(dotPrismaClientEdgeDir, 'index.js')
+    const defaultEdgeIndexDtsPath = path.join(dotPrismaClientEdgeDir, 'index.d.ts')
+    await makeDir(dotPrismaClientEdgeDir)
+
+    if (!fs.existsSync(defaultNodeIndexPath)) {
+      await copyFile(path.join(__dirname, 'default-index.js'), defaultNodeIndexPath)
+    }
+
+    if (!fs.existsSync(defaultBrowserIndexPath)) {
+      await copyFile(path.join(__dirname, 'default-index-browser.js'), defaultBrowserIndexPath)
+    }
+
+    if (!fs.existsSync(defaultNodeIndexDtsPath)) {
+      await copyFile(path.join(__dirname, 'default-index.d.ts'), defaultNodeIndexDtsPath)
     }
-    if (!fs.existsSync(defaultIndexBrowserJSPath)) {
-      await copyFile(path.join(__dirname, 'default-index-browser.js'), defaultIndexBrowserJSPath)
+
+    if (!fs.existsSync(defaultEdgeIndexPath)) {
+      await copyFile(path.join(__dirname, 'default-edge.js'), defaultEdgeIndexPath)
     }
 
-    if (!fs.existsSync(defaultIndexDTSPath)) {
-      await copyFile(path.join(__dirname, 'default-index.d.ts'), defaultIndexDTSPath)
+    if (!fs.existsSync(defaultEdgeIndexDtsPath)) {
+      await copyFile(path.join(__dirname, 'default-index.d.ts'), defaultEdgeIndexDtsPath)
```

Comment:
Would it be easier to prepare the whole directory template in `scripts` and copy it here instead of building the directory structure file by file?

(Maybe not much easier, since what's happening here is a bit more complicated than just copying, but at least it should be a more scalable approach I think)